modules/ROOT/pages/silverblue-guide.adoc

```diff
@@ -159,6 +159,136 @@ there have likely noted when they did a remote-ls command on their configured
+
+image::sfg_flathub_download.png[title="Flathub download options"]
+
+After the download is complete, a new window will open showing the Flathub
+repository install options. This window also shows the source location of the
+repository to be installed under the details heading (1). To start the
+installation of the Flathub repository, click on the `Install` button (2).
+
+image::sfg_flathub_install.png[title="Flathub install window"]
+
+After the repository install process is complete, the window will be updated to
+show a `Remove` button in place of the `Install` button. Close the window (1).
+
+image::sfg_flathub_finish.png[title="Flathub install complete"]
+
+Now, if you go back to Gnome Software you should see Flathub listed in the
+'Software Repositories' (1).
+
+image::sfg_repositories.png[title="Gnome Software repositories"]
+
+====== Installing Flatpak Application from Flathub
+Now that the Flathub repository is configured you can install some Flatpak software
+applications from Flathub. The following example shows how to install the
+LibreOffice software application Flatpak from the Flathub repository.
+
+Open the web browser (Firefox) and go to the http://flathub.org/home page. Search
+for LibreOffice and click on the link to open the LibreOffice details page. Click
+on the `Install` button (1) to start the install or see below about reviewing the
+other info on the detail page.
+
+image::sfg_libreoffice_info1.png[title="LibreOffice details page"]
+
+The details below the `Install` button include a link to a setup guide
+for LibreOffice that should be reviewed before beginning the install to see if any
+pre-installation configuration actions are required. There is also some screen shots
+of the application windows to give a preview of what the application interface looks
+like. Below the screenshots is a description of the software application.
+
+Further down the page under the "Additional Information" is information about the
+last update date for the Flatpak (1), the software version(2), and the license
+information (3). Some repositories may not have the latest software version of
+the application and some repositories contain non-free software licensed
+applications. Review this information to be sure it meets your needs. There is
+also a command line instruction to install the Flatpak (4).
+
+image::sfg_libreoffice_info2.png[title="LibreOffice Additional Information"]
+
+After clicking the `Install` button, a download information window will be shown.
+Verify the correct Flatpak is being downloaded and then click on the `OK` button
+to begin installing the LibreOffice application.
+
+image::sfg_libreoffice_install.png[title="LibreOffice Flatpak download"]
+
+Once the Flatpak is downloaded, the Gnome Software application will open a new
+window with an `Install` button (1). This window also shows details again about
+the Flatpak application (2). Notice that LibreOffice is Sandboxed. This means it
```

Comment:
'sandboxed' shouldn't be capitalized here.

Reply:
Fixed capitalization of sandboxed.